app/src/main/java/comcmput301f16t01/github/carrier/Users/UserProfileActivity.java

```diff
@@ -255,4 +281,85 @@ public void hideKeyboard(View v) {
         InputMethodManager imm = (InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE);
         imm.hideSoftInputFromWindow(v.getWindowToken(), 0);
     }
+
+
+    /**
+     * This is the function that calls the number presented
+     *
+     * @param v view used for this function
+     */
+    public void callPhone(View v) {
+        /* Source: http://stackoverflow.com/questions/5403308/make-a-phone-call-click-on-a-button
+        * Author: Shaista Naaz
+        * Retrieved on: November 21st 2016 */
```

Comment:
Remind me to fix code attribution format. (If you want to see how I've been doing it you can make the change yourself by looking at my code-reuse-fixes branch.) I can make the fix if you want once this is merged with master. Just maybe make an issue to remind me to deal with them or a TODO here or something :)